javascript/javascript1/week4/main.js

```diff
@@ -0,0 +1,132 @@
+const names = [];
+const notes = [];
+
+function dialog() {
+    const userInput = prompt("Hello there!");
+    getReply(userInput);
+}
+
+function getReply(command) {
+
+const words = command.split(" ");
+  
+if (words[0].toLowerCase() === "hello" && 
+      words[1].toLowerCase() === "my" && 
+      words[2].toLowerCase() === "name" && 
+      words[3].toLowerCase() === "is") {
```

Comment:
instead of converting to lowercase here you can do it in `const words = command.split(" ").toLowerCase();`

Reply:
Thank you! I will update my code